apps/ping_pong/app-colcon.meta

```diff
@@ -9,8 +9,7 @@
                 "-DRMW_UXRCE_MAX_SERVICES=0",
                 "-DRMW_UXRCE_MAX_CLIENTS=0",
                 "-DRMW_UXRCE_MAX_HISTORY=4",
-                "-DRMW_UXRCE_DEFAULT_SERIAL_DEVICE=3"
```

Comment:
This option gets already set when the [configure.sh](https://github.com/micro-ROS/micro-ros-build/blob/26e9b778aa5067da3578210b3ddac2bfe0086147/micro_ros_setup/config/freertos/olimex-stm32-e407/configure.sh#L36) script is run with `-t serial`.
If set here it overrides the setting of the configure script.